docs/source/guide/lre-5-theory.md

```diff
@@ -0,0 +1,87 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.11.4
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+
+```{warning}
+The user guide for LRE in Mitiq is currently under construction.
+```
+
+# What is the theory behind LRE?
+
+Layerwise Richardson Extrapolation (LRE), an error mitigation technique, introduced in
+{cite}`Russo_2024_LRE` extends the ideas found in ZNE by allowing users to create multiple noise-scaled variations of the input
+circuit such that the noiseless expectation value is extrapolated from the execution of each
+noisy circuit.
+
+Similar to [ZNE](zne.md), this process works in two steps:
+
+- **Step 1:** Intentionally create multiple noise-scaled but logically equivalent circuits by scaling each layer or chunk of the input circuit through unitary folding.
+
+- **Step 2:** Extrapolate to the noiseless limit using multivariate richardson extrapolation.
+
+LRE leverages the flexible configuration space of layerwise unitary folding,
+allowing for a more nuanced mitigation of errors by treating the noise level of each layer of
+the quantum circuit as an independent variable.
+
+## Step 1: Intentionally create multiple noise-scaled but logically equivalent circuits
+
+The goal is to create noise-scaled circuits of different depths where the layers in each circuit are scaled in
+a specific pattern as a result of unitary folding. This pattern is often described by the vector of scale factor vectors
+generated by the fold multiplier and the chosen degree for multivariate Richardson extrapolation polynomial. For more information
+on unitary folding, go to [What is the theory behind ZNE?](zne-5-theory.md).
```

Comment:
> Add a bit more of a lead in. An example where we have a simple circuit, and we show what each λ i is would be super helpful.

I have this description in part 4 of the lre user guide (not pushed yet because that's what I am currently working on).

Reply:
Ah great! In that case it probably makes sense to add a link to that section/example from this part.